tabulator-tables - selectableCheck is deprecated by selectableRowsCheck

danielHin opened the pull request:
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:

- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/olifolkerd/tabulator/commit/7e143a5b24db9c660fa7503ea98bdb75c5876f0c
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the `package.json`.

DangerBotOSS commented:
<!--
  0 failure: 
  0 warning: 
  
  1 markdown notices
  DangerID: danger-id-Danger;
-->



## Formatting

The following files are not formatted:
1. types/tabulator-tables/index.d.ts

Consider running `pnpm dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against ef4c1ad51a664c55ac3435977645d0a79089d0d9
</p>

mike-lischke commented:
@danielHin Deprecation doesn't mean it's been removed. Can you point to a place in the documentation that mentions removal of these deprecated functions? If they are still supported we also should not remove them, yet.

danielHin commented:
@mike-lischke. 
I added the selectableRowsCheck and marked selectableCheck as deprecated in the first place. But then I noticed the deprecated options not supported by tabulator > 5.6 anymore. 

I was using selectableCheck in my project and noticed, its not working anymore, I had to use selectableRowsCheck. 
 
Checked the tabulator-tables repo and noticed, the options aren't registered anymore and even the deprecation check is commented out by the maintainer. 

https://github.com/olifolkerd/tabulator/blob/df8fde15ccd5def10a6ac269db9e3f35ced9cc4b/src/js/modules/SelectRow/SelectRow.js#L18-L22

https://github.com/olifolkerd/tabulator/blob/df8fde15ccd5def10a6ac269db9e3f35ced9cc4b/src/js/modules/SelectRow/SelectRow.js#L62

Renaming was done in 5.6.0 https://github.com/olifolkerd/tabulator/commit/7e143a5b24db9c660fa7503ea98bdb75c5876f0c

So I considered it as "not supported" anymore and removed it (to avoid others also getting issues when using these options). 

Documentation is not containing any references to the deprecated options:
https://tabulator.info/docs/6.3/select

danielHin commented:
Ready to merge

danielHin commented:
@mike-lischke sorry, closed the pr without merging...
